var/spack/repos/builtin/packages/mepo/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Mepo(PythonPackage):
+    """Tool to manage (m)ultiple git r(epo)sitories"""
+
+    homepage = "https://github.com/GEOS-ESM/mepo"
+    git = "https://github.com/GEOS-ESM/mepo.git"
+    pypi = "mepo/mepo-2.0.0rc4.tar.gz"
+
+    maintainers("pchakraborty", "mathomp4")
+
+    license("Apache-2.0", checked_by="mathomp4")
+
+    version("2.0.0rc4", sha256="5f6113be565c561c08114355570a259042b25222a9e8e1dc6e6e44448381cd36")
+    version("2.0.0rc3", sha256="c0c897a33f5018489e6cc14892961831c8922a3378ac30436496c52bf877aff7")
+
+    depends_on("python@3.9:", type=("build", "run"))
+    depends_on("py-colorama", type=("build", "run"))
+    depends_on("py-pyyaml@6:", type=("build", "run"))
```

Comment:
You can now use the proper version of this after a rebase.

Reply:
@tldahlgren Okay. New question: how does one express:
```toml
    "pyyaml>=6.0.1",
```
in spack-speak? Is it: `py-pyyaml@6.0.1:`? Does that allow for `py-pyyaml@6.1`? Or even `py-pyyaml@6.0.2`?

Reply:
Ah. It does seem to work in my testing. Okay. Let me do some tests and update incoming.

Reply:
You beat me to an answer.  Yes, `@6.0.1:` covers all versions >= `6.0.1`.  If you want more information, we discuss version ranges in the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges.

Reply:
`py-pyyaml` dependency updated.